scipy/stats/tests/test_mstats_basic.py

```diff
@@ -1874,13 +1874,6 @@ def test_kstest_2samp(self):
                         res3 = stats.kstest(xm, y, alternative=alternative, mode=mode)
                         assert_equal(np.asarray(res1), np.asarray(res3))
 
-    def test_nametuples_agree(self):
```

Comment:
Rather than modifying this test, we can just remove it. It doesn't belong in `test_mstats_basic.py` if it's about `stats` objects anyway, and `Ks_2sampResult` is no longer used either in `stats` or `mstats`, so I think it's safe to remove this.